configs/projects/puppet-agent.rb

```diff
@@ -180,11 +206,19 @@
     proj.component "ruby-selinux"
   end
 
-  proj.directory proj.install_root
-  proj.directory proj.prefix
-  proj.directory proj.sysconfdir
+
+  if platform.is_windows?
+    proj.directory proj.install_root, id: proj.company_id, name: proj.company_name
```

Comment:
These look like new options for Vanagon. They'll just be ignored when calling directory?